src/components/CardTable/CardTable.jsx

```diff
@@ -85,55 +86,61 @@ export default class CardTable extends Component {
     clip: 'rect(0 0 0 0)',
   }
 
+  tableNodes = {}
+
   get deck () {
-    const { cardsPerDeck, classNames, headers } = this.props
+    const { cardsPerDeck, classNames, headers, rows } = this.props
     const {
       items: decks,
       itemIndex: deckIndex,
       minIndex,
       nextIndex,
       prevIndex,
-    } = this.itemsProperties(cardsPerDeck)
+    } = this.itemsProperties(cardsPerDeck || rows.length)
     const deckRows = decks[deckIndex]
 
     return (
       <div>
         <Deck classNames={classNames} headers={headers} rows={deckRows} />
         <div className={navigation}>
-          {this.backButton(prevIndex < minIndex, this.prevPage(cardsPerDeck))}
+          {this.backButton(prevIndex < minIndex, this.prevPage(cardsPerDeck || rows.length))}
           <div>{`${nextIndex}/${decks.length}`}</div>
-          {this.nextButton(nextIndex >= decks.length, this.nextPage(cardsPerDeck))}
+          {this.nextButton(nextIndex >= decks.length, this.nextPage(cardsPerDeck || rows.length))}
```

Comment:
`cardsPerDeck || rows.length` is being used multiple times, perhaps define it as a class getter:
```
get cardsPerDeck () {
  return this.props.cardsPerDeck || this.props.rows.length
}
```